src/components/ShareButtons/ShareButtonGroup.tsx

```diff
@@ -18,13 +18,19 @@ const ShareImageButtons: React.FC<{
   disabled?: boolean;
   onSaveImage?: () => void;
   onCopyLink?: () => void;
+  onShareOnFacebook: () => void;
+  onShareOnTwitter: () => void;
+  onShareOnLinkedin: () => void;
```

Comment:
To be a bit less repetitive, you could have `onShare: (dest: string) => void` and pass 'Facebook', 'Twitter', or 'LinkedIn' as appropriate.

If you go that route, you could optionally create a string union type to make it self-documenting / typesafe.

`type ShareDestination = 'Facebook'|'Twitter'|'LinkedIn';`

But feel free to keep it as-is.  It's a minor cleanup at best.

Reply:
Good idea! Will do on a separate PR